Makefile

```diff
@@ -8,6 +8,9 @@ setup-apple-targets:
 setup-android-targets:
 	rustup target add x86_64-linux-android aarch64-linux-android armv7-linux-androideabi i686-linux-android
 
+setup-go-targets:
+	rustup target add aarch64-apple-darwin x86_64-apple-darwin
```

Comment:
I'd think in practice most people would be running this in linux environments, right?

Reply:
Yeah, probably, I just prioritized darwin because that's what I'm running :) .